bliss/datasets/sdss.py

```diff
@@ -1,16 +1,16 @@
 import pathlib
 import pickle
 import warnings
-from math import floor, ceil
+from math import ceil, floor
```

Comment:
The `isort` program sorts all the imports. That way there's only one way to do it, so the diffs for subsequent PRs will be smaller.